enhancements/README.md

```diff
@@ -0,0 +1,23 @@
+# Enhancement Proposal
+
+## Why do we need it?
+
+Enhancement Proposal (EP) is a design document providing information to the community, or describing a new feature for Dex.
+
+We intend EPs to be the primary mechanisms for proposing major new features.
+This will make it easier for community to describe, track, and look through the history of changes that a going to affect the future development of a product.
```

Comment:
```suggestion
This will make it easier for the community to describe, track, and look through the history of changes that affected the development of the project.
```